src/promptflow-devkit/promptflow/_sdk/_service/utils/utils.py

```diff
@@ -99,13 +96,18 @@ def get_port_from_config(create_if_not_exists=False):
     return port
 
 
-def dump_port_to_config(port):
+def get_port_file_location():
     if is_run_from_built_binary():
         port_file_path = HOME_PROMPT_FLOW_DIR / PF_SERVICE_PORT_FILE
         port_file_path.touch(mode=read_write_by_user(), exist_ok=True)
     else:
-        # Set port to ~/.promptflow/pfs/**_pf.port, if already have a port in file , will overwrite it.
         port_file_path = get_current_env_pfs_file(PF_SERVICE_PORT_FILE)
+    return port_file_path
+
+
+def dump_port_to_config(port):
```

Comment:
should we make the port as a Configuration item?

Reply:
we will always start from PF_SERVICE_DEFAULT_PORT now